src/components/custom-field-input-single-choice/custom-field-input-single-choice.jsx

```diff
@@ -216,7 +103,7 @@ CustomFieldInputSingleChoice.propTypes = {
 
 CustomFieldInputSingleChoice.defaultProps = {
   choices: [],
-  className: styles.container,
+  className: '',
```

Comment:
Setting this to an empty string will override the default Select className. Is that intended?

Reply:
Noooope, definitely not! Fixing...